### DISCUSSION TITLE
"sport - eco" graphics restoration in cbd dacia sandero 2021

### DISCUSSION THREAD

**Discussion by labdomenico** [cedricp/ddt4all#1287] [Q&A] [answered]
Good evening,
dacia sandero stepway gpl 2021.
I activated the sport function and everything works properly, it is activated by pressing the "eco" button.
By mistake, I deleted the text that notifies the active mode (sport = sport; eco = ecological; normal driving = no text) and I don't know which part I changed. The "sport - eco" function continues to work properly when the button is pressed but I don't have any notifications on the cbd display.
Can you help me restore the graphics in the cbd?
Thanks everyone
![SPORT GITHUB](https://github.com/user-attachments/assets/5c719067-d6c6-4e67-a5f2-25c9bd5959d7)

**Comment by matzel82** [comment]
Automatic Configuration at Dacia Garage and a Software Update

**Comment by labdomenico** [comment]
thank you very much, but I don't want to go to the Dacia workshop. I accessed the micom menu, I activated the Renault interface and mysense on the media display. if I select "sport" from mysense the writing also appears in the CBD. if I select the writing "eco" on the CBD nothing appears. Link at the video. thank you.  https://drive.google.com/file/d/1nMbmWrrVsGwNzIOQoXVzG4gAS--Qo_BW/view?usp=drivesdk

**Comment by labdomenico** [comment, marked as answer]
Solved
![IMG_20241103_212401](https://github.com/user-attachments/assets/5056c0b5-d627-4075-b9cf-0da9f3d45e64)

**Comment by Zipizapeyuki** [comment]
> Good evening, dacia sandero stepway gpl 2021. I activated the sport function and everything works properly, it is activated by pressing the "eco" button. By mistake, I deleted the text that notifies the active mode (sport = sport; eco = ecological; normal driving = no text) and I don't know which part I changed. The "sport - eco" function continues to work properly when the button is pressed but I don't have any notifications on the cbd display. Can you help me restore the graphics in the cbd? Thanks everyone ![SPORT GITHUB](https://private-user-images.githubusercontent.com/129509776/382508025-5c719067-d6c6-4e67-a5f2-25c9bd5959d7.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMzMDUyMjUsIm5iZiI6MTczMzMwNDkyNSwicGF0aCI6Ii8xMjk1MDk3NzYvMzgyNTA4MDI1LTVjNzE5MDY3LWQ2YzYtNGU2Ny1hNWYyLTI1YzliZDU5NTlkNy5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwNFQwOTM1MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kOTlmMWVmNjQ0OGQ3MGY3YWFkMDc3ZmE3YzIxYjMyYWIzMmM0MWU4NTdmNTE0NzRhOGE3ZWE1NGMyNzcxMDg3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.jY_oVCKKs4OLSdda3zdGxAYKcot-jIWeIKahXQl1sjU)

How activate the sport Mode on dacia Sandero 3? I need tour help  please?

**Reply by labdomenico** [reply]
hi, i don't speak english well. if you want you can join the following telegram group and follow the detailed guide i made. maybe, i can help you with google translate. see you soon

https://t.me/DaciaSandero2021

**Comment by Zipizapeyuki** [comment]
Thanks

El 4 de diciembre de 2024 12:23:17 UTC, labdomenico ***@***.***> escribió:
>hi, i don't speak english well. if you want you can join the following telegram group and follow the detailed guide i made. maybe, i can help you with google translate. see you soon
>
>https://t.me/DaciaSandero2021
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/cedricp/ddt4all/discussions/1287#discussioncomment-11460778
>You are receiving this because you commented.
>
>Message ID: ***@***.***>
Dani Gimeno Cañas
